modules/hcp-import-limitations.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-import.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-import-limitations_{context}"]
+= Limitations of managing imported hosted clusters
+
+Hosted clusters are automatically imported into the local {mce}, unlike a standalone {product-title} or third party clusters. Hosted clusters run some of their agents in the _hosted mode_ so that the agents do not use the resources of your cluster. If you choose to automatically import hosted clusters, you can update hosted clusters by using the `HostedCluster` resource on the management cluster.
+
+You can import hosted clusters into a location other than the local {mce-short} by using the {mce-short} or {rh-rhacm-first}. You must use the following options to update hosted clusters:
+
+* The console or the command-line interface of the management cluster, if you use the {mce} to import a hosted cluster.
+* A policy, if you use the {rh-rhacm} hub cluster to import a hosted cluster.
+
+The {mce} console can directly access the `HostedCluster` and `NodePool` resources. {rh-rhacm} agent runs inside the managed cluster. In {hcp}, the control plane handles the updates, which is present outside the hosted clusters. Unlike hosted clusters, you can update standalone {product-title} clusters by using the `ClusterVersion` resource, which is present inside an {product-title} cluster.
```

Comment:
How can I update hostedClusters? Before talking about standalone we should clarify hosted.

Reply:
To cover the _how_ part, existing section links for the reference:
- https://docs.openshift.com/container-platform/4.17/hosted_control_planes/hcp-updating.html#hcp-update-using-mce-console_hcp-updating --- Updating a hosted cluster (i.e control plane and node pools) by using the multicluster engine Operator console
- https://83506--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating-cluster-cli#update-upgrading-cli_updating-cluster-cli -- Updating a standalone cluster by using clusterversion